ebin/rabbit_app.in

```diff
@@ -38,6 +38,7 @@
          {default_vhost, <<"/">>},
          {default_permissions, [<<".*">>, <<".*">>, <<".*">>]},
          {loopback_users, [<<"guest">>]},
+         {password_hashing_mod, rabbit_password_hashing_sha256},
```

Comment:
`backing_queue_module` has the full `module` word, so maybe use that here as well